integration/solana/token.sol

```diff
@@ -0,0 +1,29 @@
+import '../../examples/spl_token.sol';
```

Comment:
Mos people only download Solang pre-compiled binary and will not have the repository on their system. Having to download an example is too burdensome. If you want to use x86 or arm intrinsics in C/C++, you just need to include a file, without any download.
One option to solve this would be to create an installation bundle that saves this includes in an installation folder and automatically configures the environment variables, so that we can just type `solang` on our terminal to use it. What do you think of this?

Reply:
We have seen various issues/questions being raised about installing solang. It would be nice to have a better way of dealing with this, but I don't know what the answer is. Something like rustup, solana-installer, or avm (anchor version manager) might be the right answer.

Hopefully we will get to a point where there will be many files that help users write solidity on solana (or other chains), including a port of openzeppelin. How that should be distributed, I don't know. Maybe something on solana-program-library.

I don't think that asking users to copy a file into their tree is too difficult, it's just not the most elegant solution.

However, the answer to all of this is not going to be solved in this PR.